1-theory/3-api-contracts.adoc

```diff
@@ -0,0 +1,232 @@
+= API Contracts
+
+The "I" means interface, and so writing a contract for that interface is the act of writing down what that interface is going to look like, to avoid folks having to guess, or go and hunt for some documentation somewhere.
```

Comment:
```suggestion
The "I" in "API" means interface, and so writing a contract for that interface is the act of writing down what that interface is going to look like, to avoid folks having to guess, or go and hunt for some documentation somewhere.
```